SportsPlanner/ViewModels/ViewModelLocator.cs

```diff
@@ -0,0 +1,47 @@
+﻿using GalaSoft.MvvmLight.Ioc;
+using GalaSoft.MvvmLight.Views;
+using Microsoft.Practices.ServiceLocation;
+using SportsPlanner.Gateways;
+
+namespace SportsPlanner.ViewModels
+{
+    public class ViewModelLocator
+    {
+        /// <summary>
+        /// Initializes a new instance of the ViewModelLocator class.
+        /// </summary>
+        public ViewModelLocator()
+        {
+            ServiceLocator.SetLocatorProvider(() => SimpleIoc.Default);
+
+            ////if (ViewModelBase.IsInDesignModeStatic)
+            ////{
+            ////    // Create design time view services and models
+            ////    SimpleIoc.Default.Register<IDataService, DesignDataService>();
+            ////}
+            ////else
+            ////{
+            ////    // Create run time view services and models
+            ////    SimpleIoc.Default.Register<IDataService, DataService>();
+            ////}
+
+            SimpleIoc.Default.Register<MainViewModel>();
+
+            SimpleIoc.Default.Register<IEventsGateway, EventsApiGateway>();
+            SimpleIoc.Default.Register<IDialogService, DialogService>();
```

Comment:
Actually, MVVM Light has different implementation for both, iOS and Android platforms. You don't have to take care about the platform, mvvm light does it for you.